build.sbt

```diff
@@ -44,3 +61,7 @@ lazy val testing = (project in file("testing"))
       Libraries.discipline
     )
   )
+
+lazy val `circe-yaml` = (project in file (".")).settings(
+  commonSettings
+) dependsOn (snake) //aggregate (snake)
```

Comment:
Actually, come to think of it, I probably should have left the name as `circe-yaml` since we had agreed to just split out `snake` as a different asset once we had a reason to use a different backend.